I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang-11.txt

```diff
@@ -1710,99 +1819,102 @@ Internet-Draft    Opt. Impairment-Aware Topo YANG Model     October 2022
 
 
       Figure 12: OT and OTSI protection function are an integral part
-                   of the WDM-TE- node (receive diection)
+                  of the WDM-TE- node (receive direction)
 
-2.10.1.2.  OT and OTSi protection/ROADM functions are in two adjecent
+2.10.1.2.  OT and OTSi protection/ROADM functions are in two adjacent
            WDM-TE-node (remote OT)
 
-   Use case (ii)
+   This use case is based on the architecture illustrated in Figure 9
+   where the optical transponder is not part of the WDM-TE-node
+   containing the ROADM function (WDM-TE-Node-2) but is part of a
+   separate WDM-TE-node (WDM-TE-Node-1) containing one or more optical
+   transponders (remote OTs).  WDM-TE-Node-2 contains:
 
+   *  Splitter/selector protection function
 
+   *  ROADM function
 
+   Figure 13 illustrates such a network configuration in transmit/
+   forward direction showing the two WDM-TE-nodes where the protection
+   function is the optical splitter in WDM-TE-Node-2 and Figure 14
+   illustrates the same network configuration in receive/reverse
+   direction where the protection function is the optical selector in
+   WDM-TE-Node-2 selecting one of the two incoming OTSi signals and
+   switching to the other incoming OTSi signal when the optical power of
+   the selected OTSi signal drops below a pre-defined threshold.
 
 
 
-
-
-
-
-
-
-
-
-
-
-
-Beller, et al.            Expires 27 April 2023                [Page 31]
+Beller, et al.            Expires 27 April 2023                [Page 33]
 
 Internet-Draft    Opt. Impairment-Aware Topo YANG Model     October 2022
 
 
-        WDM-TE-Node 1                      WDM-TE-Node 2
+   In the network configuration shown in Figure 13 and Figure 14,
+   respectively, the two WDM-TE-nodes are interconnected via a TE-link
+   carrying a single OTSi signal.  An add-drop LTP represents the end of
+   the TE-link in WDM-TE-Node-2.  Similar to the protected TTP in
+   Section 2.10.1.1, the existing optional protection-type leaf is used
```

Comment:
In case of connectivity-matrix, we had to augment the LTP definition with a new optional leaf named protection-type.
I would suggest to reword as : 
" In this case a new optional protection-type leaf has been added under LTP of the node,  to indicate..."
The rest is correct

Reply:
@sergiobelotti, thanks for your comments!

I updated the paragraph and the following modified text should address your comment:

`In the network configuration shown in Figure 13 and Figure 14,
respectively, the two WDM-TE-nodes are interconnected via a TE-link
only carrying a single OTSi signal.  This TE-link interconnects the
remote OT with an add-drop port of WDM-TE-Node-2 and in the following
the qualifier "add-drop" is used to refer to that LTP as opposed to
the line LTPs representing degrees of WDM-TE-Node-2.  Similar to the
protected TTP in Section 2.10.1.1, the optional protection-type leaf
is used to indicate whether the add-drop LTP is connected to a
protection function and then to two line LTPs via the ROADM function
inside WDM-TE-Node-2 or whether it is connected to a single line LTP
via the ROADM function inside WDM-TE-Node-2 (unprotected add-drop
LTP).  While the protection-type attribute was already defined for
the TTP, the YANG model has been augmented to also support this
optional attribute for the LTP.`